protobuf-lite/src/main/java/io/grpc/protobuf/lite/ProtoLiteUtils.java

```diff
@@ -38,13 +38,16 @@
 import com.google.protobuf.InvalidProtocolBufferException;
 import com.google.protobuf.MessageLite;
 import com.google.protobuf.Parser;
+import com.google.protobuf.UnsafeByteOperations;
 
 import io.grpc.ExperimentalApi;
 import io.grpc.Metadata;
 import io.grpc.MethodDescriptor.Marshaller;
 import io.grpc.Status;
+import io.grpc.internal.ReadableBuffer;
```

Comment:
Using `byte[]` (with `KnownLength`) doesn't seem to need any additional interface. If you need to, use `ByteBuffer.wrap()`.